jonprl-mode.el

```diff
@@ -48,6 +48,23 @@
   :type 'file
   :group 'jonprl)
 
+(defcustom jonprl-pretty-symbols
+  '(("*" . ?×)
+   ("->" . ?→)
+   ("<>" . ?★)
+   ("=>" . ?⇒)
+   ("=def=" . ?≝)
+   ("base" . ?𝔻)
+   ("fix" . ?μ)
```

Comment:
A fancy `Y` would probably work nicely...

Well, I am surprised you have not yet added display rules to convert everything to fraktur anyway ;-)